pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -534,7 +534,7 @@ func (m *cgroupManagerImpl) Create(cgroupConfig *CgroupConfig) error {
 
 	// it may confuse why we call set after we do apply, but the issue is that runc
 	// follows a similar pattern.  it's needed to ensure cpu quota is set properly.
-	if err := m.Update(cgroupConfig); err != nil {
+	if err := manager.Set(resources); err != nil {
```

Comment:
This is a separate commit (I guess you are not reviewing this commit-by-commit, since this is the first commit).

I can certainly move it out to a separate PR and made this one a draft waiting for that other PR to be merged. Same for other 5 preparatory commits.

Reply:
> This is a separate commit (I guess you are not reviewing this commit-by-commit, since this is the first commit).

Yeah, thanks for doing that! I definitely prefer your style of doing it with separate commits with well defined commit messages! I am a huge fan of that, thanks!

 > I can certainly move it out to a separate PR and made this one a draft waiting for that other PR to be merged. Same for other 5 preparatory commits.
 
 Thanks. The only worry is doing a lot of these cleanups in the same PR as a big dependency change; mostly for being (better) able to bisect the CI running on the master branch, as well as the "work" of reverting if we have to do that. Other than for dependency changes or verrry big PRs, I do prefer your style waay over the alternative. Again, thanks for doing this!